src/amo/components/SearchFilters/index.js

```diff
@@ -112,6 +122,33 @@ export class SearchFiltersBase extends React.Component<InternalProps> {
     return false;
   };
 
+  changeColorFilter: (colorValue: string) => void = (colorValue: string) => {
```

Comment:
Personally, I find it somewhat confusing that in order to remove a selected filter I have to select that filter value again. Can we not render a clear button to clear the filter if a value is selected? This would simplify the logic for "onCcolorClick" as well as make the UX more direct.

Reply:
I found rendering a clear button more confusing because there is a "white" one already. IMHO this is not a blocker, we can iterate on it with UX later if the feature is useful/popular enough.

Reply:
Does white filter for white themes or does it reset to include all themes again? IMO the white button is even more confusing because I might think it would be a clear button but it may or may not be?

I guess the only way for a user to know if a filter value is selected is if none of them have the outline. I think this is not clear at all.

Could you not have an additional button with an X in it and no background. Tooltip says "clear" and it is rendered only if any value is selected. Where is the potential confusion?

Reply:
White filters for white themes. It's not a the beginning or end (it's after black and grey) so I don't feel it could be interpreted any other way. But if we add another it would be weird IMHO.

We could have an unselect button with a X in it, I just thought it was ugly.

Reply:
f928061820643a2e1243101483b2b048fb1762fe adds an explicit clear button that only shows up if there is a color selected. What do you think ?